api/api.proto

```diff
@@ -19,6 +19,7 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 	schemes: HTTPS;
 	consumes: "application/json";
 	produces: "application/json";
+	produces: "application/octet-stream";
```

Comment:
@aleksandrmelnikov Do we need this?

Reply:
I think yes because when an artifact file is retrieved, core-ui treats it as an "octet-stream".
- And artifact file is returned as a []byte in go.

Adding it here makes notes that our api may return an octet-stream.

Reply:
@aleksandrmelnikov Alrighty.